apps/sms/index.html

```diff
@@ -149,6 +156,33 @@ <h1 data-l10n-id="messages"></h1>
           <a id="threads-settings-button" data-icon="settings" data-l10n-id="settingsButton"></a>
         </gaia-header>
         <article id="threads-container" class="view-body edit-container" data-type="list">
+          <gaia-fast-list id="threads-list" top="0" bottom="0">
+            <!--<template header>
+              <gaia-sub-header data-time="${daySection}">${section}</gaia-sub-header>
+            </template>-->
+            <template item>
+              <li class="${classList}" data-last-message-type="${type}" data-time="${timestamp}" data-thread-id="${id}" data-draftId="${draftId}" id="thread-${id}">
+                <label class="pack-checkbox negative">
+                  <input type="checkbox" value="${id}" data-mode="${mode}">
+                  <span></span>
+                </label>
+                <a href="${hash}">
+                  <aside role="note" data-l10n-id="${iconLabel}" class="icon icon-unread icon-draft"></aside>
+                  <aside class="pack-end threadlist-item-picture"
+                         role="img" aria-label="Contact photo"
+                         data-l10n-id="contact-photo" data-show-contact="${showContact}" data-contact-photo="${photoUrl}">
+                    <span data-type="img" style="${bgImgStyle}">${titleLength}</span>
```

Comment:
Remove all this markup and replace with:

```html
<div class="image"><img/></div>
```
You can use the [image example](https://github.com/gaia-components/gaia-fast-list/tree/master/examples/images) as guidance. Note you have to define a [`getItemImageSrc()` function](https://github.com/gaia-components/gaia-fast-list/blob/master/examples/images/app.js#L5).